app/components/course/CourseCard.tsx

```diff
@@ -1,9 +1,24 @@
-import React from "react";
+import React, { useState } from "react";
 import { CourseCardProps } from "@/app/model/CourseInterface";
 import Link from "next/link";
 import CourseCardDropDown from "./CourseDropDownButton";
 
-const CourseCard = ({ title, courseCode, tag }: CourseCardProps) => {
+const CourseCard = ({
+  title,
+  courseCode,
+  tag = [],
+  properties = {},
+  offered = {},
+  prerequsits = [],
+  status = "No Selection"
+}: CourseCardProps) => {
+  const [state, setState] = useState(status);
+
+  const offeredSemesters = [];
+  if (offered.fall) offeredSemesters.push("Fall");
+  if (offered.spring) offeredSemesters.push("Spring");
+  if (offered.summer) offeredSemesters.push("Summer");
+
   return (
     <section className="course-card">
       <div className="flex flex-col fold:flex-row justify-between items-center">
```

Comment:
The status button is inconsistent. Change items-center to items-start

Reply:
Fixed